light-rest-4j/src/main/java/com/networknt/codegen/rest/OpenApiGenerator.java

```diff
@@ -233,6 +233,9 @@ public void generate(String targetPath, Object model, Any config) throws IOExcep
                     String classVarName = key;
                     String modelFileName = key.substring(0, 1).toUpperCase() + key.substring(1);
                     //System.out.println("props = " + Any.wrap(props));
+                    if (!"object".equals(type)) {
+                        continue;
+                    }
```

Comment:
Could you add a comment to this check explaining why it's necessary

Reply:
Sure, I will do taht